apps/scan/frontend/src/app_root.tsx

```diff
@@ -40,12 +40,14 @@ import { SystemAdministratorScreen } from './screens/system_administrator_screen
 import { ScannerCoverOpenScreen } from './screens/scanner_cover_open_screen';
 import { PrinterCoverOpenScreen } from './screens/printer_cover_open_screen';
 import { ScannerDoubleFeedCalibrationScreen } from './screens/scanner_double_feed_calibration_screen';
+import { useVoterSettingsControls } from './utils/use_voter_settings_controls';
 
 export function AppRoot(): JSX.Element | null {
   const [
     shouldStayOnCastVoteRecordSyncRequiredScreen,
     setShouldStayOnCastVoteRecordSyncRequiredScreen,
   ] = useState(false);
+  const voterSettingsControls = useVoterSettingsControls();
```

Comment:
probably rename this to something like `useScanVoterSettings`, because im wondering if "controls" makes it seem optional, but it is actually managing required state regardless if the controls are used